### DISCUSSION TITLE
NIP57 (Lightning Zaps)

### DISCUSSION THREAD

**Discussion by avlo** [tcheeric/nostr-java#163] [General]
Hi, @tcheeric /All.  I'm inclined to complete [NIP57 (Lightning Zaps)](https://nostr-nips.com/nip-57) implementation following your established NIP-api ([NIP57](https://github.com/tcheeric/nostr-java/blob/develop/nostr-java-api/src/main/java/nostr/api/NIP57.java)), NIP-impl and NipXXEvent design and implementation pattern(s).

I didn't see any branches related to NIP57 nor discussions regarding the same and just wanted to make sure another/preferred implementation is not already underway before starting.

**Comment by tcheeric** [comment]
Hi Nick, please go ahead, no one else is working on it. Thanks! 
-- 
 Sent with Tuta; enjoy secure & ad-free emails: 
 https://tuta.com


18 May 2024, 03:01 by ***@***.***:
